tokio-console/src/view/task.rs

```diff
@@ -155,10 +155,16 @@ impl TaskView {
             Span::raw(task.target()),
         ]));
 
-        overview.push(Spans::from(vec![
-            bold("Location: "),
-            Span::raw(task.location()),
-        ]));
+        let title = "Location: ";
+        let mut m = Span::raw(task.location());
```

Comment:
For style reasons, I'd skip the `mut` here and assign the location from an `if .. else` expression. Something like:

```rust
let location = if row_len <= stats_area[0].width.into() {
    Span::raw(task.location())
} else {
   // existing logic to produce: Span::raw(format!("...{}", location));
}

overview.push(Spans::from(vec![bold(title), location]));
```

We have a preference for more descriptive variable names too. So ideally `m` would be `location` or perhaps `value` (as it goes with `title`).

Reply:
I improved the syntax!